### PR TITLE
reorganize release notes for 22.1.0

### PR DESCRIPTION
**Pull request by JelleZijlstra**

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -2,49 +2,59 @@
 
 ## Unreleased
 
-### _Black_
+### Highlights
 
+- At long last, _Black_ is no longer a beta product! This is the first non-beta release
```
Comment (felix-hilden) [inline CHANGES.md:7]: Instead of an item this could be text under the top heading.

**Review by felix-hilden — APPROVED** [review]
Looks good! Just one opinion below.

We could use these headings as a template for future changelogs to reduce conflicts until we decide on which tool to use in #2766.

File: CHANGES.md [hunk 2]
```diff
@@ -2,49 +2,59 @@
...
+  and the first release covered by our new stability policy.
```
Comment (JelleZijlstra) [inline CHANGES.md:8]:
```suggestion
At long last, _Black_ is no longer a beta product! This is the first non-beta release
and the first release covered by our new stability policy.

### Highlights

```
Sure

File: CHANGES.md [hunk 3]
```diff
@@ -2,49 +2,60 @@
 
 ## Unreleased
 
-### _Black_
+At long last, _Black_ is no longer a beta product! This is the first non-beta release
```
Comment (JelleZijlstra) [inline CHANGES.md:5]:
```suggestion

At long last, _Black_ is no longer a beta product! This is the first non-beta release
```

**Review by isidentical — APPROVED** [review]

**Comment by JelleZijlstra** [comment]
I'll hold off on merging this for a while to keep the merge conflict situation manageable.

**Comment by JelleZijlstra** [comment]
> It feels weird to have a section for such a minor change

I don't feel strongly about this, just made the section because the changelog entry was there and didn't fit anywhere else. Plus I'd generally err on the side of documenting more. Especially with sections like this, it's easy for people to skip over things they're not interested in.

File: CHANGES.md [hunk 4]
```diff
@@ -2,49 +2,60 @@
...
   `values: Tuple[int, ...] = 1, 2, 3`) now implies 3.8+ (#2708)
-- Text coloring added in the final statistics (#2712)
-- For stubs, one blank line between class attributes and methods is now kept if there's
-  at least one pre-existing blank line (#2736)
-- Verbose mode also now describes how a project root was discovered and which paths will
-  be formatted. (#2526)
 - Speed-up the new backtracking parser about 4X in general (enabled when
   `--target-version` is set to 3.10 and higher). (#2728)
 - Fix handling of standalone `match()` or `case()` when there is a trailing newline or a
   comment inside of the parentheses. (#2760)
```
Comment (JelleZijlstra) [inline CHANGES.md:35]:
```suggestion
- Fix handling of standalone `match()` or `case()` when there is a trailing newline or a
  comment inside of the parentheses. (#2760)

### Performance

- Speed-up the new backtracking parser about 4X in general (enabled when
  `--target-version` is set to 3.10 and higher). (#2728)
```
This section can also be used for mypyc

**Review by ichard26 — APPROVED** [review]
This looks so much better, thank you!

We should add an entry about mypyc though, not sure whether to put in under highlights or performance (I'd lean towards performance but mypyc could break things although I sure hope not). 
> \- \_Black\_  is now compiled with \[mypyc](https://github.com/mypyc/mypyc) for an overall 2x speed-up. (#1009, #2431)

File: CHANGES.md [hunk 5]
```diff
@@ -2,54 +2,66 @@
...
 - Unparenthesized tuples on annotated assignments (e.g
   `values: Tuple[int, ...] = 1, 2, 3`) now implies 3.8+ (#2708)
+- Fix handling of standalone `match()` or `case()` when there is a trailing newline or a
+  comment inside of the parentheses. (#2760)
+
+### Performance
+
+- Speed-up the new backtracking parser about 4X in general (enabled when
+  `--target-version` is set to 3.10 and higher). (#2728)
+
```
Comment (ichard26) [inline CHANGES.md:39]:
```suggestion
- _Black_ is now compiled with [mypyc](https://github.com/mypyc/mypyc) for an overall 2x
  speed-up. (#1009, #2431)

```
for ease of application

Comment (JelleZijlstra) [reply CHANGES.md:39]: Agree, it should be under "performance". Maybe we should say something about the OSs for which we'll have wheels?

Comment (ichard26) [reply CHANGES.md:39]: Updated.

Comment (JelleZijlstra) [reply CHANGES.md:42]: Thanks!

**Pull request merged**